pkgs/development/libraries/gegl/default.nix

```diff
@@ -65,6 +56,7 @@ stdenv.mkDerivation rec {
     vala
     gobject-introspection
     gtk-doc
+    gi-docgen
```

Comment:
Does this not replace gtk-doc? Do the new docs properly go to devdoc output?

Reply:
From my understanding, both are required to build the package.

Reply:
Looking at the outputs the both docs appear to cover the same API.  And `gtk-doc` is actually [disabled by default](https://gitlab.gnome.org/GNOME/gegl/-/commit/239e060d71fc3a668374efecb0b173eee67eb381#ca9bb7eff80503c97c83505e8acea4002fd87ac6_10_10) so we should probably remove it (along with the docbook dependencies).

And the devdoc should be moved to devdoc output either way.